plugins/examples/custom-suggester/src/main/java/org/elasticsearch/example/customsuggester/CustomSuggester.java

```diff
@@ -35,15 +36,12 @@
         String name,
         CustomSuggestionContext suggestion,
         IndexSearcher searcher,
-        CharsRefBuilder spare) {
-
-        // Get the suggestion context
-        String text = suggestion.getText().utf8ToString();
+        CharsRefBuilder spare) throws IOException {
 
         // create two suggestions with 12 and 123 appended
-        CustomSuggestion response = new CustomSuggestion(name, suggestion.getSize(), "suggestion-dummy-value");
-
-        CustomSuggestion.Entry entry = new CustomSuggestion.Entry(new Text(text), 0, text.length(), "entry-dummy-value");
+        CustomSuggestion response = emptySuggstion(name, suggestion, spare);
```

Comment:
fox typo :)

Reply:
It wasn't me, what did I do??? :)
Cannot find the error unfortunately...

Reply:
emptySuggstion missing e, should be emptySuggestion ?

Reply:
Thanks, I was 🙈